datacommons/setup.py

```diff
@@ -23,15 +25,10 @@
 URL = 'https://github.com/datacommonsorg/api-python'
 EMAIL = 'support@datacommons.org'
 AUTHOR = 'datacommons.org'
-REQUIRES_PYTHON = '>=2.7'
+REQUIRES_PYTHON = '>=3.7'
 VERSION = '1.4.3'
-
-REQUIRED = [
-    'six',
-]
-
+REQUIRED = ['six', 'requests']
```

Comment:
do we still need six if we're fully on py 3?

Reply:
Will remove six after the old code is deleted.